examples/validating-public-input/aligned-integration/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "aligned-integration"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+aligned-sdk = { git = "https://github.com/yetanotherco/aligned_layer", tag="v0.3.0" }
```

Comment:
Need to update to sdk 0.4.0 here (and update code to get nonce). Check https://github.com/yetanotherco/aligned_layer/pull/728 for changes

Reply:
Solved in [87dee6d](https://github.com/yetanotherco/aligned_layer/pull/648/commits/87dee6d2fa5d60a56d34a719553d34e99c8df9be)